submissions/mayowaShoyinka/javaScript/index.js

```diff
@@ -1,8 +1,8 @@
 var ProductList = [
```

Comment:
Is there a reason you modified this file?

Reply:
if i understand your question well, are you asking the reason why i changed the n<=w to n<=dif
if so, its because in the loop i already used dif which means to differentiate (its the stopping criteria for my loop.

please let me know if i answered your question well. Thanks

Reply:
but if you are talking about the second code, i changed it because, i feel the work is continuous: and since the task was based on product-name, product-URL and product-price which we worked with in our css task (product-page). That's why. Thanks